includes/demo-poem.php

```diff
@@ -7,6 +7,11 @@
  * @package    Poetwp\Includes
  */
 
+/**
+ * Includes the Demo Poem class file.
+ */
+require_once POETWP_DIR . 'includes/class-demo-poem.php';
```

Comment:
Keep all file loading in the root plugin file - that's where this code is bootstrapped, and it's simpler to find it all in one place (in the future, an autoloader can be swapped in).